### ISSUE TITLE
Runtime panic on .onion domain

### ISSUE THREAD

**Issue by vladtreny** [zonedb/zonedb#886]
This will create runtime panic
```

z := zonedb.PublicZone("anything.onion")
z.WhoisServer()


```

This library stings eyes from onions. 🥲

**Comment by case** [comment]
This may have been addressed in #914, which added `.onion` to ZoneDB along with the other special-use zones.

**Comment by ydnar** [comment]
`PublicZone` may return `nil`. It is the caller’s responsibility to check for `nil`.
